users/views.py

```diff
@@ -67,4 +69,44 @@ def post(self, request):
             return JsonResponse({'message':'LOGIN_SUCCESS','token':access_token,'user_name':user.nickname}, status=200)
 
         except KeyError:
-            return JsonResponse({'message':'KEY_ERROR'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+    
+class LikeView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            product_id = data['product_id']
+        
+            if not Product.objects.filter(id = product_id ).exists():
+                return JsonResponse({'message':'INVALID_PRODUCT'}, status=401)
+ 
+            if not Like.objects.filter(
+                user_id = request.user.id,
+                product_id = product_id).exists():
```

Comment:
```suggestion
            if not Like.objects.filter(user=user, product_id = product_id).exists():
```
- 굳이 줄을 바꾸지 않아도 될 것 같아요. 오히려 줄을 바꿔서 가독성이 떨어지는 것 같습니다.
- request.user로 user을 알 수 있기 때문에 user_id 대신에 user로 바로 받아도 됩니다.

Reply:
한줄로 바꾸시라고 한 것은 filter()안에 user=user등 값들이 2개 밖에 없어서 한줄로 봐도 한번에 볼 수 있어서 바꾸라고 한건지 궁금합니다.
(만약에 여러개의 매개변수들이 들어간다고 한다면 줄바꿈을 사용하는 것이 좋은것이겠죠??)

user=request.user를 사용해서 user = user.id를 만드니깐 
ValueError: Cannot assign "13": "Like.user" must be a "User" instance. 라는 오류가 떠서 그대로 user_id를 그대로 사용하였습니다.